docs/404.html

```diff
@@ -69,11 +69,10 @@
             <li data-nav-id="/sdk-cubic/" class="dd-item">
             <a href="/sdk-cubic/"><i class="fa fa-fw fa-home"></i></a>
             </li>
-    <li data-nav-id="/sdk-cubic/getting-started/" class="dd-item alwaysopen haschildren
+    <li data-nav-id="/sdk-cubic/getting-started/" class="dd-item haschildren
```

Comment:
Why remove `alwaysopen`?

Reply:
Before, we only had the two section called `Getting started` and `Cubic API Reference`, and Getting started was the main section, so it made sense to always have it expanded in the side bar.  
Now we have added another section that is equally important, it didn't look as good to keep this one always open.  The sections will be expanded whenever you are in a page that it contains.